bcda/api.go

```diff
@@ -242,26 +241,8 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
 
-	i, err := strconv.Atoi(jobID)
-	if err != nil {
-		log.Print(err)
-		oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.Processing)
-		responseutils.WriteError(oo, w, http.StatusBadRequest)
-		return
```

Comment:
I think we should keep this check on the job ID. We know `/jobs/abc` is a bad request, so we shouldn't query the database with that value.

Reply:
Nvm, I see this moved to middleware.